### PR TITLE
Abb95

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New ms.

### CODE REVIEW

File: ParisBNF/abb/BNFabb95.xml [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
+ <desc type="Correction">
+An omitted word is written in the upper margin, in a crude hand, its place in the text is indicated with
+<foreign xml:lang="am">tamallas</foreign>-sign.
+</desc>
+</item>
+
+ <item xml:id="e14">
+ <locus target="#97v"/>
+  <desc type="Unclear">
+ An unclear note in the gut margin, written in a crude hand.
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb95.xml:355]: what is the gut margin? :)

File: ParisBNF/abb/BNFabb95.xml [hunk 2]
```diff
@@ -0,0 +1,483 @@
...
+                        Talismanic drawing of a cross and intersecting curved lines
+                        with loops at the end (stylized monograpms?).<desc>
+                      </desc>
+                     </decoNote>
+               </decoDesc>
+                  <additions>
+                    <list>
+                      <item xml:id="a1">
+                        <locus target="#1r"/>
+                        <desc type="Unclear">The note is Pater Noster in Coptic Bohairic transcribed in
```
Comment (thea-m) [inline ParisBNF/abb/BNFabb95.xml:217]: Maybe the ID could be used, what do you think? Especially as people might also search for "Our Father" and won't find it then

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you! Note the small corrections with locus and ref I carried out :)

File: ParisBNF/abb/BNFabb95.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb95.xml:355]: Margin along the centrefold.

File: ParisBNF/abb/BNFabb95.xml [hunk 3]
```diff
@@ -214,7 +219,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     <list>
                       <item xml:id="a1">
                         <locus target="#1r"/>
-                        <desc type="Unclear">The note is Pater Noster in Coptic Bohairic transcribed in
+                        <desc type="GuestText" corresp="ETH1292Bohair">Pater Noster in Coptic Bohairic transcribed in
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb95.xml:222]: something wrong here
ETH1292Bohair should be deleted completely, we only use ETH-records for ethnic/language groups
for the language we need an xml:lang type
for the text the record LIT2168PaterN with a title iID for the version n Bohairic (e.g. LIT2168PaterN#Bohairic) should be used

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb95.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb95.xml:222]: OK

Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb95.xml:222]: Attempted

File: ParisBNF/abb/BNFabb95.xml [hunk 4]
```diff
@@ -219,7 +219,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                     <list>
                       <item xml:id="a1">
                         <locus target="#1r"/>
-                        <desc type="GuestText" corresp="ETH1292Bohair">Pater Noster in Coptic Bohairic transcribed in
+                        <desc type="GuestText" corresp="LIT2168PaterN">Pater Noster in Coptic Bohairic transcribed in
```
Comment (eu-genia) [inline ParisBNF/abb/BNFabb95.xml:222]: LIT2168PaterN#t4coptic

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ParisBNF/abb/BNFabb95.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ParisBNF/abb/BNFabb95.xml:222]: OK, thanks!

**Review by eu-genia — APPROVED** [review]

**Review by thea-m — APPROVED** [review]

**Pull request merged**